src/web/app.py

```
@@ -57,153 +59,450 @@
-                if not token.isspace():
-                    await asyncio.sleep(config.WEB_STREAMING_DELAY)
-
-        # Send citations after answer completes
-        yield f"event: citations\ndata: {json.dumps({'citations': enriched_citations})}\n\n"
+    return combined
+
+
+async def generate_streaming_response(
+    query: str,
+    session_id: str,
+    history: Optional[List[dict]] = None
```

Ruff (0.14.6)

Unused function argument: `history`

(ARG001)